CONFIG.md

```diff
@@ -275,6 +275,55 @@ directly to
 [grunt-contrib-compass](https://github.com/gruntjs/grunt-contrib-compass)
 for this theme.
 
+### Proxying Grunt Tasks
+
+Many themes come with their own Grunt integration. In those cases you can
+configure Grunt-Drupal-Tasks for awareness of that and remove the theme
+from most handling.
+
+In order to exclude your theme from handling, specify the `proxy` property in
```

Comment:
If we wanted to support gulp or a shell script, wouldn't we want to use a different property name, like `grunt` vs. `gulp` vs. `shell`?